src/utils/folder-utils.ts

```diff
@@ -83,15 +86,22 @@ export const setPaths = (): void => {
     ? yarleOptions.outputDir
     : `${process.cwd()}${path.sep}${yarleOptions.outputDir}`;
 
-  paths.mdPath = `${outputDir}${path.sep}notes${path.sep}`;
+  
+  paths.mdPath = `${outputDir}${path.sep}notes${path.sep}`; 
```

Comment:
it looks to be different because of the extra space at the end? hmm... tslint should do it's job in this case i think. please remove it.